app/views/bookings/new.html.erb

```diff
@@ -13,6 +13,10 @@
   <p>
     <%= f.label :house, 'Select a place:' %>
     <%= f.select :house_id, options_from_collection_for_select(House.ordered, :id, :name), include_blank: true %>
+  </p>
+  <p>
+    <%= f.label :numberOfGuests, 'Add the number of guests:' %>
```

Comment:
a ruby-s nevezéktanban snake_case -t szoktunk használni, itt is kéne működnie vele. kérlek nézd meg, hogy a labelre kattintva tényleg elkezd-e a mezőbe írni